regression/multi-svc-app/multi_svc_app_suite_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package multi_svc_app_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aws/copilot-cli/regression/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	toCLI   *client.CLI
+	fromCLI *client.CLI
+
+	appName string
+)
+
+// The Addons suite runs creates a new application with additional resources.
+func TestMultiSvcApp(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Regression For Multi Svc App Suite")
+}
+
+var _ = BeforeSuite(func() {
+	var err error
+
+	toCLI, err = client.NewCLI(os.Getenv("REGRESSION_TEST_TO_PATH"))
+	Expect(err).NotTo(HaveOccurred())
+
+	fromCLI, err = client.NewCLI(os.Getenv("REGRESSION_TEST_FROM_PATH"))
+	Expect(err).NotTo(HaveOccurred())
+
+	appName = fmt.Sprintf("regression-%d", time.Now().Unix())
```

Comment:
I don't remember our app name char limit, but should we add an indication that this is for multisvcapp:
```suggestion
	appName = fmt.Sprintf("regression-mutisvcapp-%d", time.Now().Unix())
```